### PR TITLE
{devel,vis}[GCCcore/11.3.0] Brotli v1.0.9, fontconfig v2.14.0, freetype v2.12.1, ...

### PR DESCRIPTION
**Pull request by jfgrimm**
depends on:

- [x] #15428 

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node077.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b78c13617fdb759fd736d6eb70dc0988 for a full test report.

File: easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.96-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.13.96'
```
Comment (branfosj) [inline easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.96-GCCcore-11.3.0.eb:4]: There is a `2.14.00` available. All the `2.13.9X` are described as `2.14 RC6`. Should we update?

Comment (jfgrimm) [reply easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.96-GCCcore-11.3.0.eb:4]: good point, I'll update

Comment (branfosj) [reply easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.96-GCCcore-11.3.0.eb:4]: Can you delete the `2.13.96` from this PR?

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node077.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e92bbb73110d2bb7e9f4cf9e82e3c569 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/35938b24298c4b3b4ab8ad247631b914 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 5 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c2aa3d35101e59d3e5f89108e3fdb671 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0306u19a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 4 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/23330e0737b3d1129914efb46332fc00 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/939ea71e47261aee50b32828becff5e7 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**